pages/index.tsx

```diff
@@ -24,57 +27,149 @@ export default function Home() {
       />
 
       <main>
-
         <HeroBanner {...content.heroBanner} aria-label={content.heroBanner.ariaLabels?.ctaButton} />
 
-        <SectionContainer id={content.ourMission.id}>
-          <PaperSection>
-            <Grid container id={`${content.ourMission.id}-main-grid`}>
-              <Grid item xs={12}>
-                <Typography variant="h2" sx={{ textAlign: 'center' }}>
-                  {content.ourMission.header}
-                </Typography>
-                <MuiMarkdown>{content.ourMission.body}</MuiMarkdown>
-                <br />
-                <br />
-                <MuiMarkdown>{content.ourMission.body2}</MuiMarkdown>
-              </Grid>
+        <SectionContainer>
+          <Grid container justifyContent="space-between" id={`${content.background.id}-main-grid`}>
+            <Grid item xs={12} md={7}>
+              <Typography
+                variant="h2"
+                sx={{ lineHeight: '40px' }}
+              >
+                {content.background.title}
+              </Typography>
+              <MuiMarkdown overrides={{
+                p: {
+                  component: Typography,
+                  props: {
+                    variant: 'body2',
+                    fontSize: '18px',
+                    lineHeight: '22px',
+                  },
+                },
+              }}
+              >
+                {content.background.body}
+              </MuiMarkdown>
+            </Grid>
+            <Grid
+              item
+              xs={12}
+              md={4}
+              display="flex"
+              justifyContent="center"
+            >
+              <Box
+                sx={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  justifyContent: 'space-evenly',
+                  alignItems: 'center',
+                  bgcolor: '#003256',
```

Comment:
Please use theme file for all these hard coded hex codes, and do so for the entire file, I won't leave the comment on each one.

Reply:
Updated hex codes in index and playcard file